packages/rum/test/viewCollection.spec.ts

```diff
@@ -308,6 +308,71 @@ describe('rum view measures', () => {
     jasmine.clock().uninstall()
   })
 
+  it('should have a loadEventEnd loading time when having no activity', () => {
+    jasmine.clock().install()
+    expect(getRumEventCount()).toEqual(1)
+    expect(getViewEvent(0).measures).toEqual({
+      errorCount: 0,
+      longTaskCount: 0,
+      resourceCount: 0,
+      userActionCount: 0,
+    })
+
+    lifeCycle.notify(
+      LifeCycleEventType.PERFORMANCE_ENTRY_COLLECTED,
+      FAKE_NAVIGATION_ENTRY as PerformanceNavigationTiming
+    )
+
+    expect(getRumEventCount()).toEqual(1)
```

Comment:
why do we need to recheck that?

Reply:
?

Reply:
`expect(getRumEventCount()).toEqual(1)` and `expect(getRumEventCount()).toEqual(2)` test that the notified event triggers a view update. It allows to test the new value of `loadingTime` being sure that an update have been completed before.

Reply:
current state is ok, in previous state you were checking it twice